spec/factories_spec.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
```

Comment:
I'd recommend using `FactoryBot.lint` instead, which also handles checking all the traits.

I've often done this as a Rake task that's part of the default CI since factories change *relatively* slowly which means running them every time (since it by definition is going to involve database hits) takes a not negligible amount of time. More info and examples here: https://github.com/thoughtbot/factory_bot/blob/master/GETTING_STARTED.md#linting-factories

Reply:
https://github.com/department-of-veterans-affairs/vets-api/pull/3345